woudc_data_registry/models.py

```diff
@@ -1137,8 +1137,9 @@ class PeerDataRecord(base):
     station_name_id = Column(String,
                              ForeignKey('station_names.station_name_id'),
                              nullable=False)
-    stn_type_enum = Enum('land', name='stn_type')
-    station_type = Column(stn_type_enum, nullable=False, default='land')
+    stn_type_enum = Enum('STN','SHP','land', name='stn_type')
```

Comment:
Use just `land`, for broad interoperability of peer data records.

Reply:
For the time being, should I set station_type to 'land' for NDACC files if the corresponding WOUDC station is type 'STN'? As of now I'm querying the database for this property for NDACC files.

Reply:
Yes, `land` is good for now (so if you derive `STN` from the WOUDC db query, set as `land` in the peer record table).